photon-api/src/main/scala/com/linkedin/photon/ml/function/glm/GLMLossFunction.scala

```diff
@@ -33,14 +33,15 @@ object GLMLossFunction {
    */
   def buildFactory(
       lossFunction: PointwiseLossFunction,
-      treeAggregateDepth: Int): (CoordinateOptimizationConfiguration) => ObjectiveFunction =
+      treeAggregateDepth: Int,
+      interceptIndex: Int = -1): (CoordinateOptimizationConfiguration) => ObjectiveFunction =
```

Comment:
Can you add a comment for this parameter?

Reply:
In addition, parameter name and type should change (see below):
```
interceptIndexOpt: Option[Int] = None
```